api/namex/resources/name_requests/report_resource.py

```diff
@@ -264,23 +265,32 @@ def _is_modernized(legal_type):
 
     @staticmethod
     def _is_colin(legal_type):
-        colin_list = ['CR', 'UL', 'CC', 'XCR', 'XUL', 'RLC']
+        colin_list = ['XCR', 'XUL', 'RLC']
         return legal_type in colin_list
     
     @staticmethod
     def _is_society(legal_type):
         society_list = ['SO', 'XSO']
         return legal_type in society_list
 
+    @staticmethod
+    def _is_ia(legal_type):
+        ia_list = ['CR', 'UL', 'CC']
+        return legal_type in ia_list
```

Comment:
Or maybe it makes more sense to label this something else, since it's not actually telling us whether this is an IA.

Maybe "is_potential_colin_company" (or similar)?

Reply:
Updated